tests/installation/online_repos.pm

```diff
@@ -62,6 +63,9 @@ sub run {
     # Test online repos dialog explicitly
     if (get_var('DISABLE_ONLINE_REPOS')) {
         disable_online_repos_explicitly;
+    } elsif (installwithaddonrepos_is_applicable() && !get_var("LIVECD")) {
```

Comment:
with same reasoning ( function is called only here ) I woud move `!get_var('LIVECD')` inside a function. in current implementation  it is not really clear why `!get_var('UPGRADE')` is part of function and  `!get_var('LIVECD')` is not

Reply:
Same reason as above, function is used in manin_common.pm.

Reply:
and in main common you also have exactly same line actually 
`installwithaddonrepos_is_applicable() && !get_var("LIVECD")` 
so why `!get_var("LIVECD")` is outside this function , for me it is still looks like it shoppuld be inside

Reply:
Well, it was already there in such state in `main_common` and I didn't want to touch what is not necessary.